plugins/php/php_fpm_status

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
I think, you are not using any bash-specific features. Thus it would be great, if you could switch to `sh` instead.